app/helpers/rad_common/application_helper.rb

```diff
@@ -259,6 +259,14 @@ def rad_toast_data(template_locals)
         controller: 'toast' }
     end
 
+    def check_blob_validity(blob)
```

Comment:
refactor this with the controller, it's doing the same thing and the logic should be shared

Reply:
we should also add a spec to /shared if possible that ensures this raise happens on each project